src/solvers/subspace_expansions/subspace_expansion.jl

```diff
@@ -0,0 +1,149 @@
+#=
```

Comment:
This file can be removed if we agree to focus this PR on the 2s subspace expansion.

Reply:
Is this for global expansion? I think it would be better to narrow the scope of this PR to 2-site expansion.